Formula/werf.rb

```diff
@@ -0,0 +1,59 @@
+class Werf < Formula
+  desc "Consistent delivery tool for Kubernetes"
+  homepage "https://werf.io/"
+  url "https://github.com/werf/werf/archive/refs/tags/v1.2.87.tar.gz"
+  sha256 "f008304325e96e6c7f28f68d49ffcc6e36ba26631d5a6bcd808c1b15d1d7c0d1"
+  license "Apache-2.0"
+  head "https://github.com/werf/werf.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on :macos
```

Comment:
Can you add a comment here to say that Linux support just hadn't been added yet? Since it doesn't actually depend on macOS

Reply:
good call.